CHANGELOG.md

```diff
@@ -4,12 +4,84 @@
 
 ### Breaking
 
+- Upgrade `react-cookie` to the latest version. @sneridagh @robgietema
+  See https://docs.voltocms.com/upgrade-guide/ for more information.
```

Comment:
Do we want to use that URL or the temporary one?

https://6.dev-docs.plone.org/volto/upgrade-guide/index.html